tests/Feature/Controllers/TasksControllerTest.php

```
@@ -0,0 +1,218 @@
+        /** Assert */
+        $response->assertOk();
+        $response->assertViewIs('projects::tasks_form');
+        $response->assertViewHas('task');
+        $response->assertViewHas('projects');
+        $response->assertViewHas('task_statuses');
+
+        /** Verify new task instance is passed */
+        $task = $response->viewData('task');
+        $this->assertInstanceOf(Task::class, $task);
+        $this->assertFalse($task->exists);
+    }
```

PHPMD (2.15.0)

The method it_displays_task_create_form is not named in camelCase. (undefined)

(CamelCaseMethodName)